contracts/issuer-type/src/entry.rs

```diff
@@ -1,25 +1,101 @@
-use core::result::Result;
-use alloc::{vec, vec::Vec};
-
 use ckb_std::{
-    debug,
-    high_level::{load_script, load_tx_hash},
+    ckb_constants::Source,
     ckb_types::{bytes::Bytes, prelude::*},
+    high_level::{load_cell_data, load_input, load_script},
+};
+use core::result::Result;
+use script_utils::{
+    class::CLASS_TYPE_ARGS_LEN,
+    error::Error,
+    hash::blake2b_160,
+    helper::count_cells_with_type_args,
+    issuer::{Issuer, ISSUER_TYPE_ARGS_LEN},
 };
 
-use crate::error::Error;
+enum Action {
+    Create,
+    Update,
+    Destroy,
+}
 
-pub fn main() -> Result<(), Error> {
+fn parse_issuer_action(args: &Bytes) -> Result<Action, Error> {
+    let check_args_equal = |type_args: &Bytes| type_args[..] == args[..];
+    let inputs_count = count_cells_with_type_args(Source::Input, &check_args_equal);
+    let outputs_count = count_cells_with_type_args(Source::Output, &check_args_equal);
 
-    let script = load_script()?;
-    let args: Bytes = script.args().unpack();
-    debug!("script args is {:?}", args);
+    if inputs_count > 1 || outputs_count > 1 || (inputs_count == 0 && outputs_count == 0) {
```

Comment:
can `inputs_count` and `outputs_count` be zero at the same time?

Reply:
No. If so, the tx dose not have issuer cells

Reply:
so, why need to check `(inputs_count == 0 && outputs_count == 0)` ?